### PR TITLE
Pre-release v2.0

### PR DESCRIPTION
**Pull request by tgiphil**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -125,13 +125,13 @@ for:
     - sh: sudo apt-get install -y git wget qemu qemu-system qemu-system-x86
 
     before_build:
-    - sh: dotnet restore $APPVEYOR_BUILD_FOLDER/Source/Mosa.LinuxCI.sln
+    - sh: dotnet restore $APPVEYOR_BUILD_FOLDER/Source/Mosa.sln
```
Comment (charsleysa) [inline appveyor.yml]: This should be changed back to use `Mosa.Linux.sln` as the normal solution uses windows-only components that can't be compiled on linux.

File: appveyor.yml [hunk 2]
```diff
@@ -125,13 +125,13 @@ for:
...
 
     build: off
 
     build_script:
-    - sh: msbuild /verbosity:minimal $APPVEYOR_BUILD_FOLDER/Source/Mosa.LinuxCI.sln
+    - sh: msbuild /verbosity:minimal $APPVEYOR_BUILD_FOLDER/Source/Mosa.sln
```
Comment (charsleysa) [inline appveyor.yml]: This should be changed back to use `Mosa.Linux.sln` as the normal solution uses windows-only components that can't be compiled on linux.

**Comment by charsleysa** [comment]
BTW, some of the linux unit tests are getting stuck with a failure to start the engine. A max retry (possible of 3) should be implemented to stop the unit tests getting stuck in a retry loop. I stopped them after being stuck for over 2 hours.

**Comment by tgiphil** [comment]
The code was updated to abort after 10 failed re-tries to start the engine.

File: appveyor.yml (hunk 2 above)
Comment (tgiphil) [reply appveyor.yml]: Good catch. The script was updated.

Separately, once all the git hub action are ported, appveyor will be discontinued.

**Comment by tgiphil** [comment]
will re-attempt again later.

**Pull request closed without merging**